vm/src/exceptions.rs

```diff
@@ -63,12 +63,8 @@ impl PyBaseException {
     }
 
     #[pyslot]
-    pub(crate) fn tp_new(
-        cls: PyTypeRef,
-        args: FuncArgs,
-        vm: &VirtualMachine,
-    ) -> PyResult<PyRef<Self>> {
-        PyBaseException::new(args.args, vm).into_ref_with_type(vm, cls)
```

Comment:
The error happened in ImportError. I think that's because `.into_ref_with_type(vm, cls)` is removed here so it lost its type .